cgi-bin/CATS/UI/Console.pm

```diff
@@ -624,16 +621,23 @@ sub console_frame
 
     console_content;
     $t->param(is_team => $is_team);
+    my $lvparams = $t->{vars};
     my $cc = $t->output;
+    my $user_filter = url_param('uf') || '';
 
     my $lv = init_console_template('console.html.tt');
+
+    $t->param($_ => $lvparams->{$_}) for qw(i_units i_values display_rows page pages);
+    $t->param($_ => $lv->settings->{$_}) for keys $lv->settings;
```

Comment:
1) Why is the first line needed when we have the second?
2) Missing dereference.